style.css

```diff
@@ -41,9 +65,10 @@ div.navbar {
 }*/
 
 footer {
-   margin-top: 50px;
+   margin-top: 550px;
```

Comment:
Why are you setting the footer with such a high margin. It would be better if it would just follow the content, no?